website/content/docs/enterprise/audit-logging.mdx

```diff
@@ -8,9 +8,11 @@ description: >-
 # Audit Logging
 
 <EnterpriseAlert>
-  This feature requires{' '}
-  <a href="https://www.hashicorp.com/products/consul/">Consul Enterprise</a>{' '}
-  with the Governance and Policy module.
+  This feature is available in v1.8.0+ of: (1) {' '}
+  <a href="https://cloud.hashicorp.com/pricing/consul/">HashiCorp Cloud Platform (HCP) Consul - Standard tier and above</a>
```

Comment:
@pcmccarron : is this the link we want to use for the link circled in red below? Or some other HCP link?

![image](https://user-images.githubusercontent.com/85913323/145654548-d9a69036-af33-4c83-91ee-8c2ba9d1f11c.png)

Reply:
Per the comment above, we should probably just link to the matrix on the CE

Reply:
@trujillo-adam: anything else you'd change about this?

![image](https://user-images.githubusercontent.com/85913323/155187022-92479cbe-7630-4c51-8bf6-560533bd0f09.png)